src/test/python/agents/atba/atba.py

```diff
@@ -33,6 +33,24 @@ def get_output_vector(self, game_tick_packet):
         if turn == -1.0:
             self.send_quick_chat(QuickChats.CHAT_EVERYONE, QuickChats.Information_IGotIt)
 
+        car_render_location = [my_car.Location.X, my_car.Location.Y, my_car.Location.Z]
+        ball_render_location = [game_tick_packet.gameball.Location.X,
+                                game_tick_packet.gameball.Location.Y,
+                                game_tick_packet.gameball.Location.Z]
+        text_render_strX = 'x:' + str(game_tick_packet.gameball.Location.X)
+        self.renderer.begin_rendering()
```

Comment:
Why expose begin_rendering and end_rendering to the bot? Can't the bot manager handle that, and just no-op during end_rendering if the bot never called any draw functions?

Reply:
I would rather have the bot makers have control over what group they want to render in case they want multiple groups that can asynchronously render.